neofetch

```diff
@@ -119,27 +119,44 @@ getdistro() {
     case "$os" in
         "Linux" )
             if type -p lsb_release >/dev/null 2>&1; then
+                distro="$(lsb_release -sd 2>/dev/null)"
```

Comment:
Personally, I don't think so.

When `distro_shorthand` is set to `tiny`/`on`, the `distro` variable from outside the case will be replaced with the one inside the case. (because it's `distro=` not `distro+=`.)

Granted, I haven't tested this personally, so I can't say it for sure.